rpcs3/rpcs3qt/gui_settings.h

```diff
@@ -213,7 +213,11 @@ namespace gui
 	const gui_save rsx_geometry = gui_save(rsx, "geometry", QByteArray());
 	const gui_save rsx_states   = gui_save(rsx, "states",   QVariantMap());
 
+#ifdef __APPLE__
+	const gui_save m_currentStylesheet = gui_save(meta, "currentStylesheet", NativeStylesheet);
```

Comment:
This makes no sense at all. If you really need to change the default, just do:
```suggestion
	const gui_save m_currentStylesheet = gui_save(meta, "currentStylesheet", "native (macOS)");
```